### ISSUE TITLE
"Unexpected Function value as type specifier" when enumerable property is added to Object.prototype

### ISSUE THREAD

**Issue by yg-i** [Koromix/koffi#111]
This doesn't affect usage at all but I thought worth reporting since I'm not sure if this is intended behavior. 

The behavior in question is that koffi.struct() throws an error if an enumerable property has been added to Object.prototype.

Error message:

```
const POINT = koffi.struct('POINT', {
                    ^
TypeError: Unexpected Function value as type specifier, expected string or type

```

Example code:
```
import koffi from 'koffi'

Object.defineProperty(globalThis.Object.prototype, 'mm', {
	value: function(){},
	writable: false,
	configurable: false,
	enumerable: true, // setting this to false and the error will no longer occur
})


const lib = koffi.load('user32.dll')

// Type declarations
const POINT = koffi.struct('POINT', {
	x: 'long',
	y: 'long',
})

// Functions declarations
const GetCursorPos = lib.func('int __stdcall GetCursorPos(_Out_ POINT *pos)')

// Get and show cursor position
let pos = {}
if (!GetCursorPos(pos)) throw new Error('Failed to get cursor position')
console.log(pos)

```

**Comment by Koromix** [comment]
Thanks, this will be fixed in the next release :)

**Comment by Koromix** [comment]
I've released Koffi 2.6.8 which contains this fix.

See the [Koffi 2.6.8 changelog entry](https://koffi.dev/changelog#koffi-2-6-8-2023-11-24)
